packages/nx-plugin/src/executors/cli/executor.ts

```diff
@@ -30,29 +32,30 @@ export default async function runAutorunExecutor(
   const { dryRun, verbose, command } = mergedOptions;
   const commandString = createCliCommandString({
     command,
+    bin,
     args: cliArgumentObject,
   });
+
   if (verbose) {
     logger.info(`Run CLI executor ${command ?? ''}`);
-    logger.info(`Command: ${commandString}`);
   }
-  if (dryRun) {
-    logger.warn(`DryRun execution of: ${commandString}`);
-  } else {
-    try {
-      await executeProcess({
-        ...createCliCommandObject({ command, args: cliArgumentObject }),
-        ...(context.cwd ? { cwd: context.cwd } : {}),
-      });
-    } catch (error) {
-      logger.error(error);
-      return {
-        success: false,
-        command: commandString,
-        error: error as Error,
-      };
-    }
+
+  try {
+    await executeProcess({
+      ...createCliCommandObject({ command, args: cliArgumentObject, bin }),
+      ...(context.cwd ? { cwd: context.cwd } : {}),
+      env,
+      dryRun,
+    });
+  } catch (error) {
+    logger.error(error);
+    return {
+      success: false,
+      command: commandString,
+      error: error as Error,
```

Comment:
I realize this wasn't introduced in this PR, but it's been bugging me :sweat_smile: Casting the error is unsafe, and given we have our `stringifyError` function, safe error handling should be easy:

```suggestion
      error: error instanceof Error ? error : new Error(stringifyError(error)),
```